### DISCUSSION TITLE
"Говорящие" псевдонимы для наборов значений и систем кодирования

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#232] [General]
Дискуссия создана по итогу обсуждения на [общей встрече 5 сентября 2023](https://fhir.ru/meetings.2023-09-05)

А.Звягин предлагает ввести человеко-узнаваемые алиасы систем кодирования.
 
Например, по МКБ10:
- алиас вальюсета $nsi-vs-idc10
- вальюсет: NsiVsIdc10
- алиас кодовой системы: $nsi-cs-idc10
- кодовая система  NsiCsIdc10

При этом, сейчас в примерах Виталия по алиасу артефакты не опознать:

Alias: $nsi-vs-1005 = http://fhir.ru/core/vs/nsi-vs-1-2-643-5-1-13-13-11-1005
ValueSet: NsiVs12643511313111005
Id:       nsi-vs-1-2-643-5-1-13-13-11-1005
Title: "Международная статистическая классификация болезней и проблем, связанных со здоровьем (10-й пересмотр) [все значения]"

Alias: $nsi-cs-1005 = https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.11.1005
CodeSystem: NsiCs12643511313111005
Id:         nsi-cs-1-2-643-5-1-13-13-11-1005
Title:     "Международная статистическая классификация болезней и проблем, связанных со здоровьем (10-й пересмотр)"

**Comment by Vrodvi** [comment]
Во-первых, у вас опечатка - idc10 - должно быть ICD-10
![image](https://github.com/fhir-ru/core/assets/92928137/c8c5359f-79da-425d-a24a-217d19357a38)

Во-вторых, МКБ-10 достаточно узнаваемая аббревиатура для неё легко подобрать алиас, а что с другими не такими очевидными названиями справочников - тоже будет по-английски подбирать названия и делать из них аббревиатуры? Это будет хоть кому-то узнаваемо? Почему тогда у вас не латиницей написаны русские термины, например, почему icd-10, а не mkb-10?

У меня алиас сделан чисто технически - последние цифры oid'а. Это простое правило формирования алиаса, а вот какое правило для более узнаваемых алиасов предлагается взамен - я пока не вижу.
Прошу сформулировать чёткий регламент присвоения алиасов - какими критериями должен руководствоваться автор при подборе алиаса?

**Comment by Vrodvi** [comment]
Кстати, я проверил - алиасы могут быть и на русском языке.
Просьба учесть это при написании регламента.
![image](https://github.com/fhir-ru/core/assets/92928137/50db9ee7-26c3-4e4e-a5df-753113ba241e)
